src/traces/parcoords/lines.js

```diff
@@ -0,0 +1,368 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var createREGL = require('regl');
+var glslify = require('glslify');
+var vertexShaderSource = glslify('./shaders/vertex.glsl');
+var fragmentShaderSource = glslify('./shaders/fragment.glsl');
+
+var depthLimitEpsilon = 1e-6; // don't change; otherwise near/far plane lines are lost
```

Comment:
Good advice, I moved such constants out of `parcoords.js` into `constants.js`. I spared these few constants in `lines.js` though, b/c  these values are being accessed in tight loops - I could still do a bunch of `var whatever = config.whatever;` but might not be worth it; also, they're rather fundamental values not meant for reconfiguration. Pls. tell me if you think it'd still be useful.